### DISCUSSION TITLE
How to set the devServer `compress` option more reasonably?

### DISCUSSION THREAD

**Discussion by Lancernix** [webpack/webpack#16979] [Q&A]
Hi, everyone. When use `webpack-dev-server` to start a react project in development env.  The `compress` option of devServer should be set `true` or `false`? Which one will have better performance?
![图片](https://user-images.githubusercontent.com/26547236/231822992-4fdf02f9-5d33-48b4-9f81-5a507d6827ed.png)

**Comment by alexander-akait** [comment]
I reccomend to use default value, and don't set it if you really don't need it

**Reply by Lancernix** [reply]
Thanks for your answer.  I would like to know when to set it to `false` and when to set it to `true`. Could you explain it briefly? 🙏

**Reply by alexander-akait** [reply]
It enableds/disables compression, by default almost all servers use it, https://en.wikipedia.org/wiki/Gzip

**Reply by nyngwang** [reply]
@alexander-akait Does enabling/disabling the compression affect the loading speed of the client's browser? I was thinking that while gzip should speed up the download, the browser may need extra time to unzip? Anyway, just interested in the details.

**Reply by alexander-akait** [reply]
gzip is fast
